code/modules/clothing/masks/gasmask.dm

```diff
@@ -136,6 +136,142 @@
 	item_state = "monkeymask"
 	flags_cover = MASKCOVERSEYES
 	resistance_flags = FLAMMABLE
+	actions_types = list(/datum/action/item_action/screech)
+	var/recent_uses = 0
+	var/broken_hailer = 0
+	var/gorilla = FALSE
+	var/cooldown_special
+
+
+/obj/item/clothing/mask/gas/monkeymask/ui_action_click(mob/user, action)
+	screech()
+
+/obj/item/clothing/mask/gas/monkeymask/attack_self()
+	screech()
+/obj/item/clothing/mask/gas/monkeymask/emag_act(mob/user as mob)
+	if(!gorilla)
+		to_chat(user, "<span class='warning'>You silently turn on [src]'s gorilla module with the cryptographic sequencer.</span>")
+	else
+		return
+
+/obj/item/clothing/mask/gas/monkeymask/verb/screech()
+	set category = "Object"
+	set name = "Screech"
+	set src in usr
+	if(!isliving(usr))
+		return
+	if(!can_use(usr))
+		return
+	if(broken_hailer)
+		to_chat(usr, "<span class='warning'>\The [src]'s screeching system is jammed full of bananas paste.</span>")

```

Comment:
"bananas paste"

Reply:
Banana should be singular, if the comment above me is unclear

Reply:
oops